frontend/src/Context/PublisherProvider/usePublisher/usePublisher.tsx

```diff
@@ -76,8 +77,8 @@ const usePublisher = (): PublisherContextType => {
   const [isPublishing, setIsPublishing] = useState(false);
   const publisherOptions = usePublisherOptions();
   const [isForceMuted, setIsForceMuted] = useState<boolean>(false);
-  const [isVideoEnabled, setIsVideoEnabled] = useState(!!publisherOptions.publishVideo);
-  const [isAudioEnabled, setIsAudioEnabled] = useState(!!publisherOptions.publishAudio);
+  const [isVideoEnabled, setIsVideoEnabled] = useState(false);
+  const [isAudioEnabled, setIsAudioEnabled] = useState(false);
```

Comment:
What's the reasoning behind setting -- or correcting, rather -- the state later, as opposed to initializing it with the correct value first?  Just curious...

Reply:
ah the reason is that I changed the publisher options to potentially be null so it was not liking that